src/main/java/net/minecraftforge/event/OnDatapackSyncEvent.java

```diff
@@ -40,8 +40,15 @@ public PlayerList getPlayerList()
      *         such as when the reload command runs.
      */
     @Nullable
-    public ServerPlayer getPlayer()
-    {
+    public ServerPlayer getPlayer() {
         return this.player;
     }
+
+    /**
+     * @return A list of players that should receive data during this event, which is the specified player (if not null)
+     *         or all players otherwise.
+     */
+    public List<ServerPlayer> getRelevantPlayers() {
```

Comment:
I would still like to see this renamed to `getPlayers()` given our discussion about it earlier.